runtime/dancebox/src/lib.rs

```diff
@@ -794,9 +794,13 @@ impl RemoveParaIdsWithNoCredits for RemoveParaIdsWithNoCreditsImpl {
             let free_session_credits = pallet_services_payment::CollatorAssignmentCredits::<Runtime>::get(para_id)
                 .unwrap_or_default();
 
+            // Should have enough tip for 2 sessions
+            let max_tip = pallet_services_payment::MaxTip::<Runtime>::get(para_id).unwrap_or_default() * 2 ;
```

Comment:
We probably need to revisit this because of the following assumptions:
- We are charging immediately this value (and therefore, we dont have to check 2 sessions but just one).
- It does not even make sense to check for couple of sessions as the container owner can change the maxTip in the middle to something higher or lower
- I think the same thing applies to the session payment itself.

What do you think?

Reply:
I have gone ahead and changed the logic a bit. Now it only checks if the tank has enough balance for only 1 session for both blocks and assignment + tip, unless the para has been assigned collators on the current session. If so, it checks that it has enough to pay for two sessions of block production